railties/lib/rails/generators/rails/plugin/plugin_generator.rb

```diff
@@ -406,6 +406,10 @@ def email
         end
       end
 
+      def rails_version_specifier(gem_version = Rails.gem_version)
+        [">= #{gem_version}"]
+      end
```

Comment:
This overrides [`Rails::Generators::AppBase#rails_version_specifier`](https://github.com/rails/rails/blob/56d8ff83726bb6c0b1f570deb057835f90a75e30/railties/lib/rails/generators/app_base.rb#L288-L299).